### ISSUE TITLE
Search for gulp in node_modules/.bin/ or other paths?

### ISSUE THREAD

**Issue by Gert-dev** [benjaminRomano/gulp-manager#17]
Hello

I was wondering if it would be possible to search for the gulp command somewhere else than in your path. Some environments install it via npm development dependencies to `node_modules/.bin/gulp`. I believe this is always the same path as the node_modules folder is not configurable IIRC. It would be neat if this folder could be automatically checked for an instance of gulp.

Another way would be to have project-specific settings, i.e. via the project-manager package, which provides a service that allows you to access these settings. That way someone could configure the path to his or her gulpfile and gulp instance on a per-project basis (I also do that in php-integrator for PHP settings).

Thanks in advance

**Comment by benjaminRomano** [comment]
I agree that would be a nice feature to have. I currently don't have the time to implement this but it should be relatively easy.

Here's how it might be achieved if you are interested in making a PR.

On this line, set the command to something like "{project_dir}/node_modules/.bin/gulp" if it exists otherwise default to "gulp". Disclaimer, I'm not 100% if this would work, but it seems like it should.

https://github.com/benjaminRomano/gulp-manager/blob/master/lib/gulpfile-runner.coffee#L28
